test/common/protobuf/utility_test.cc

```diff
@@ -1358,6 +1357,125 @@ TEST_F(ProtobufUtilityTest, ValueUtilLoadFromYamlObjectWithIgnoredEntries) {
       "struct_value { fields { key: \"baz\" value { string_value: \"qux\" } } }"));
 }
 
+TEST_F(ProtobufUtilityTest, ValueToJson) {
+  {
+    // null
+    ProtobufWkt::Value v;
+
+    std::string json_string;
+    ValueUtil::toJsonString(v, json_string);
+    EXPECT_EQ(json_string, "null");
+    json_string.clear();
+
```

Comment:
I would add a test-method helper that takes a Value as an arg and returns the json string, with these 4 lines. Then you can structure tests like

```
  ProtobufWkt::Value v;
  EXPECT_EQ("null", toJson(v));
  v.set_null_value(ProtobufWkt::NULL_VALUE));
  EXPECT_EQ("null", toJson(v));
```

I think this will make tests more information-dense and easier to read.